src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MemberNameCheck.java

```diff
@@ -56,16 +56,74 @@
  * <pre>
  * &lt;module name="MemberName"/&gt;
```

Comment:
Please replace quotes with `&quot;`

Reply:
What happens if no escaping done ?

Reply:
So do I need to change this?

Reply:
> What happens if no escaping done ?

I do not know. But it will be better if we use one style per file.

Reply:
My personal preference is to avoid escaping if possible, as it is made code less readable.

@rnveach , what is your position on this ?
It is better to make decision on this for future updates.

In this PR, let's escape, to be consistent.

Reply:
Done.

Reply:
I have no real preference.
@pbludov has been escaping these in all his PRs and I mentioned in one if he wants this done everywhere, he should make it part of the test otherwise new committers won't do it.

> What happens if no escaping done ?

Nothing as long as we escape the `<`s are escaped. These were real `"` for the longest time. You only really need to escape if you are using it inside another quote like an attribute or text area or such.

Reply:
@pbludov,

>I do not know.

Does it make sense to do massive search-replace to un escape double braces ?